fastlane/Snapfile

```diff
@@ -0,0 +1,50 @@
+# Uncomment the lines below you want to change by removing the # in the beginning
+
+# A list of devices you want to take the screenshots from
+# devices([
+#   "iPhone 8",
+#   "iPhone 8 Plus",
+#   "iPhone SE",
+#   "iPhone X",
+#   "iPad Pro (12.9-inch)",
+#   "iPad Pro (9.7-inch)",
+#   "Apple TV 1080p"
+# ])
+
+# XXX do we want more devices?
+devices([
+    "iPhone 8",
+    "iPhone X"
+])
+
+# XXX we should generate this from ooni/translations
+languages([
```

Comment:
well it's in a different repo.
for now the languages are "ar ca de el es fa fr hi it pt_BR tr sq zh_TW"

Reply:
Yeah. When you run `update_language_ios.sh` it should write to this file and also to the `fastlane/en/title.strings` for the text to include in the screenshots.

Reply:
makes sense. let's do it after we merge this PR maybe